portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,21 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Jacinda's Portfolio</h1>
+      <p>Welcome to my portfolio.</p>
+      <img src = "/images/tenor.gif" />
+      <p>Hi everyone! My full name is Jacinda Ryzel Soto. Right now you are visiting
+        my portfolio page which is kinda cool. I like to code (if that wasn't obvious), read,
+        listen to music, paint, and more!</p>
+      <p>Here is a funny picture of me.</p>
+      <img src = "/images/patrick.gif" />
```

Comment:
Lol. This could be even funnier if you make it a link that people have to click on (like the ones you have below) in order to see it, so they don't realize they've been had at first. (Not a mandatory change or anything, I just think the idea is funny.)